internal/database/webhooks.go

```diff
@@ -402,6 +407,7 @@ func scanWebhook(sc dbutil.Scanner, key encryption.Key) (*types.Webhook, error)
 		&dbutil.NullString{S: &keyID},
 		&dbutil.NullInt32{N: &hook.CreatedByUserID},
 		&dbutil.NullInt32{N: &hook.UpdatedByUserID},
+		&dbutil.NullString{S: &hook.Name},
```

Comment:
It's not a `NullString`, though, right? 
```suggestion
		&hook.Name,
```

Reply:
sure, nice catch, thanks! used to be, but not now